driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -3096,6 +3107,10 @@ public String toString() {
                 }
               }
               break;
+            case REMOVED:
```

Comment:
OK I see you added this case branch that was somehow missing since the first Cloud API. But I also see that a case branch for `ADDED` existed before the Cloud API:

```
case ADDED:
              Host newHost = metadata.newHost(address);
              Host previous = metadata.addIfAbsent(newHost);
              if (previous == null) {
                futures.add(schedule(hostAdded(newHost)));
              } else if (!previous.isUp()) {
                futures.add(schedule(hostUp(previous)));
              }
              break;
```

Shouldn't we trigger a `submitNodeListRefresh()` for it? Or is it because `ADDED` is not being triggered anymore? (But in this case, maybe we can delete the `HostEvent.ADDED` enum constant.)

Reply:
> is it because ADDED is not being triggered anymore?

Yes, that.

You're right, I'll remove the unused constant.